index.html

```diff
@@ -381,6 +381,9 @@ <h3>
         <p>A <a>PerformanceNavigationTiming</a> has an associated
         {{NavigationType}} <a data-dfn-for="PerformanceNavigationTiming"><dfn>navigation type</dfn></a>.
 
+        <p>A <a>PerformanceNavigationTiming</a> has an associated null or [=service worker timing info=]
+        <a data-dfn-for="PerformanceNavigationTiming"> <dfn>service worker timing</dfn></a>.
```

Comment:
```suggestion
        <p>A {{PerformanceNavigationTiming}} has an associated null or [=service worker timing info=] 
        <dfn data-dfn-for="PerformanceNavigationTiming">service worker timing</dfn>.
```

This seems wrong tho... shouldn't "service worker timing" get defined by "service worker timing info" (or be pulled from either the SW spec or from Fetch?)

Reply:
[=service worker timing info=] is pulled from the SW spec. What's wrong?